pkg/cmd/reset.go

```diff
@@ -17,6 +17,11 @@ limitations under the License.
 package cmd
 
 import (
+	"context"
+	"fmt"
+	"github.com/kubermatic/machine-controller/pkg/apis/cluster/v1alpha1"
```

Comment:
This import should also be moved to a dedicated group. You can move it to a new group below `k8c.io` imports, like here:

https://github.com/kubermatic/kubeone/blob/a21344b6b32458cf088efb1d71a1a55c0507b685/pkg/templates/machinecontroller/machines.go#L19-L38